<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Seleccione una clase y haga click en el plano para ingresar un patr칩n de entrenamiento</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="plot_area"/>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Datos de entrenamiento</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="2">
           <widget class="QLineEdit" name="learning_rate">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0.02</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="iteration_label">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1" colspan="2">
           <widget class="QPushButton" name="begin_button">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Iniciar</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QTableWidget" name="result_table">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>75</number>
            </attribute>
            <row>
             <property name="text">
              <string>N1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>N2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>N3</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>N4</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>N5</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>MSE</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>w1</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>w2</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>b</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="0" column="2">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="1" column="2">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="1" column="3">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="2" column="1">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="2" column="2">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="2" column="3">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="3" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="3" column="1">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="3" column="2">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="3" column="3">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="4" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="4" column="1">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="4" column="2">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="4" column="3">
             <property name="text">
              <string/>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="max_iterations">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>200</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Error objetivo</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Raz칩n de aprendizaje</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="target_error">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0.08</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Iteraciones m치ximas</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1" colspan="2">
           <widget class="QPushButton" name="clear_button">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Limpiar</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Iteraci칩n</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
